README.md

```diff
@@ -132,6 +131,28 @@ extism call plugin.wasm sum --input='{"a": 20, "b": 21}' --wasi
 # => {"sum":41}
 ```
 
+You can also specify your input and output types as dataclasses using `extism.Json`:
+
+```python
+from typing import Optional, List 
+from dataclasses import dataclass
+
+# ...
+
+@dataclass
+class User(extism.Json):
+  admin: bool
```

Comment:
@zshipko does there exist a way to provide the keyname if it differs from the property name?

Reply:
no problem, just checking